src/tests/SimpleScreenshot.test.tsx

```diff
@@ -32,7 +32,7 @@ describe('Simple Screenshot Test', () => {
     ];
     
     // Convert the simple trajectory to timeline entries
-    const timelineEntries = convertOpenHandsTrajectory(simpleTrajectory);
+    const timelineEntries = createMockTimelineEntries(simpleTrajectory);
```

Comment:
@OpenHands, I think we should just remove all "timeline entries" related code in the repo since we are not using them at all